widgetssdk/src/main/java/com/glia/widgets/config/components/Text.java

```diff
@@ -0,0 +1,41 @@
+package com.glia.widgets.config.components;
+
+import androidx.annotation.Nullable;
+
+import com.glia.widgets.config.components.base.Alignment;
+import com.google.gson.annotations.SerializedName;
+
+public class Text {
+    @SerializedName("foreground")
+    @Nullable
+    private RemoteColor foreground;
```

Comment:
NIT: not sure if it would be better or actually worse. But just as an option for discussion. Maybe we can rename it to `fontColor`. And the `background` to `backgroundColor`?

Reply:
I renamed to `textColor` and `backgroundColor`